lib/fluent/plugin_helper/cert_option.rb

```diff
@@ -46,12 +46,10 @@ def cert_option_server_validate!(conf)
         case
         when conf.cert_path
           raise Fluent::ConfigError, "private_key_path is required when cert_path is specified" unless conf.private_key_path
-          raise Fluent::ConfigError, "private_key_passphrase is required when cert_path is specified" unless conf.private_key_passphrase
```

Comment:
How about logging warn message like "For security reason, setting private_key_passphrase is recommended"?